### ISSUE TITLE
T: Theme and topic template - Modèle de thème et de sujet

### ISSUE THREAD

**Issue by quidampepin** [canada-ca/design-system-systeme-conception#23]
Alpha prototypes are here:
https://tbs-proto1.openplus.ca/en/landing/working-for-government
https://tbs-proto1.openplus.ca/en/government/publicservice/pay.html

**Comment by cfarquharson** [comment]
I thought we were getting rid of the doormat text (the blurb below the H1) in the new topic patterns?

**Comment by quidampepin** [comment]
@cfarquharson you're talking about the description right before the Most requested? We haven't started documenting the guidance on this (I'm hoping to start today), but I was thinking of making that text optional (the image as well would be optional). There may be situation where it's warranted to add that description, IMO - but there are situations where it would not be needed.

**Comment by cfarquharson** [comment]
@quidampepin yeah, that's the text I'm talking about.  I'm OK if it's optional, I just don't want it required :)

**Comment by cfarquharson** [comment]
@quidampepin 
Also, can https://tbs-proto1.openplus.ca/en/landing/working-for-government double as a 'campaign' page?  I think the graphic space is inline with a campaign's desire for that extra-something-special, while freeing up the content area to be 'back-to-business'.

**Comment by quidampepin** [comment]
As as I'm concerned, the same type of design could be used in a campaign page, yes. Also, by allowing some graphic treatment on these pages, we're hoping that the temptation to use a campaign page would be reduced...

**Comment by pcwsmith** [comment]
@quidampepin @cfarquharson IA considerations to this line of thinking? we have always wanted to keep campaigny stuff outside of the main theme & topic structure...

**Comment by pcwsmith** [comment]
looping in @patlaj

**Comment by cfarquharson** [comment]
@pcwsmith I'm not sure that the design impacts the IA of it, as it's just a visual treatment to the H1 container.   This design, stemming from the homepage of Canada.ca seems to be flexible enough to be repurposed beyond the homepage, as it's now the institutional profile, and the 'decorated' topic page.  I'm just suggesting that we firm it in as the design-of-choice for any page that needs that extra oomph.

**Comment by cfarquharson** [comment]
@quidampepin Can you add in the sign-in/register button to the patterns too?  I believe you have them placed top-right in other patterns

**Comment by quidampepin** [comment]
@pcwsmith in my mind, this is a bit like the Spotlight pattern. It allows "some" visual treatment to satisfy the appetite for this - so campaign pages could be used only when there is a real campaign (so: some kind of promo, a temporary page for it, a call to action, etc.). Sometimes, the campaign page has been used just to circumvent the lack of visual treatment on topic pages, for example. This i what we want to avoid. 

@cfarquharson you mean: add the sign-in button to the list of patterns to work on (not necessarily strictly on the theme/topic template, right?)

**Comment by cfarquharson** [comment]
@quidampepin Yep, that's what I mean for the sign-in :)

**Comment by quidampepin** [comment]
Done!

**Comment by quidampepin** [comment]
Draft guidance: https://design.canada.ca/mandatory-templates/theme-topic-alpha.html 
Very draft coded layout (missing some custom CSS): https://design.canada.ca/coded-layout/theme_topic_guidance.html

**Comment by quidampepin** [comment]
Latest prototypes:
THEME: https://tbs-proto1.openplus.ca/en/landing/working-for-government 
TOPIC: https://tbs-proto1.openplus.ca/en/government/publicservice/pay.html

**Comment by ecoops** [comment]
I get that this is supposed to be about the visual design, but why did you also change the content to remove military tasks from the Public service and military theme?
On the theme page, you removed 
- "military" from the title, 
- "Benefits and services for the military" from Services and Information, and 
- "Military Pay" from Most requested.

Despite them being some of the most clicked items on the page, and without consulting DND or the TBS theme leads (Véro). Why?

<img width="1170" alt="Public service and military theme" src="https://user-images.githubusercontent.com/61590339/76228348-fcd6b500-61f6-11ea-8c5c-794bc16c57f2.png">

**Comment by quidampepin** [comment]
@ecoops: I agree that using this example may have been a bit clumsy. The point was really about the design, not the content. That specific page comes from an optimization project we did with the Public service pay information (where we experimented with using "Working for the government instead of "Public service and military", and it did work better for public servants...). Vero's team was indeed involved in the project. But we're not saying right now to change the name of the theme (because of course it would no longer work for military). In our current project, we're also testing some tasks with veterans. Maybe (and I really say MAYBE - it wouldn't happen without working with DND!) a new audience/theme called "Veterans and military" (and keeping "Working for the government" for public servants) would work better. But this wouldn't happen without testing, consulting, etc. Bottom line: we are NOT advising or requesting any change to these pages at the moment. It was just to show, in the open, the latest version of the prototype.

**Comment by quidampepin** [comment]
The template is now in Beta:

- https://design.canada.ca/mandatory-templates/theme-topic.html 
- https://conception.canada.ca/modeles-obligatoire/theme-sujet.html
